src/Host.cpp

```diff
@@ -178,10 +178,11 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
     // file auditing and other issues)
     mErrorLogStream.setDevice(&mErrorLogFile);
 
-    // There was a map load attempt made here but it did not seem to be needed
-    // and it caused issues being doing in the constructor (some other classes
-    // were not fully initialised at this point) so it seemed sensible to remove
-    // it - Slysven
```

Comment:
Oops! Thanks for the catch. It was not intended that a new profile that doesn't have the map gets auto-prompted to download one - it's only if there was a map present that it should be loaded.

I'll fix this.